docs/diagrams/MenuFeaturesClassDiagram.puml

```diff
@@ -1,7 +1,8 @@
 @startuml
 'https://plantuml.com/class-diagram
 
-!include Style.puml
```

Comment:
Consider incorporating Style.puml in v2.1 so that diagrams in the DG have a more consistent look